Mathlib/SetTheory/Cardinal/Basic.lean

```diff
@@ -301,34 +294,74 @@ we provide this statement separately so you don't have to solve the specializati
 theorem lift_mk_eq' {α : Type u} {β : Type v} : lift.{v} #α = lift.{u} #β ↔ Nonempty (α ≃ β) :=
   lift_mk_eq.{u, v, 0}
 
-@[simp]
-theorem lift_le {a b : Cardinal.{v}} : lift.{u, v} a ≤ lift.{u, v} b ↔ a ≤ b :=
-  inductionOn₂ a b fun α β => by
-    rw [← lift_umax]
-    exact lift_mk_le.{u}
+/-- `Cardinal.lift` as an `InitialSeg`. -/
+@[simps!]
+def lift.initialSeg : @InitialSeg Cardinal.{u} Cardinal.{max u v} (· < ·) (· < ·) := by
```

Comment:
The name here matches `Ordinal.lift.initialSeg`, but I could name this `liftInitialSeg` instead and change the former's name in a separate PR.

Reply:
I agree that `liftInitialSeg` is a better name. Can you change?

Reply:
I'd rather deprecate `Ordinal.lift.initialSeg` first, I do that in #16995.